vscode/microsoft-kiota/src/steps.ts

```diff
@@ -168,7 +169,7 @@ export async function generateSteps(existingConfiguration: Partial<GenerateState
             case 'plugin':
                 return inputPluginName;
             case 'apimanifest':
-                return inputManifestName;
+                return inputPluginType;
```

Comment:
Is this case 'apimanifest' still or 'others'? From AC it looks like the question will change from label `Generate an API manifest` to label `others`. Do we still intend to keep the generationType for this label as `apimanifest` so that we still switch to case `apimanifest` or is this changing to generationType 'others' so that we change to case `others`?

Reply:
We'll follow the AC. As it has recently been clarified